ddtrace/llmobs/_integrations/langchain.py

```diff
@@ -194,6 +203,50 @@ def _llmobs_set_meta_tags_from_chain(
             except TypeError:
                 log.warning("Failed to serialize chain output data to JSON")
 
+    def _llmobs_set_meta_tags_from_embedding(
+        self,
+        span: Span,
+        input_texts: Union[str, List[str]],
+        output_embedding: Union[List[float], List[List[float]], None],
+        error: bool = False,
+        is_workflow: bool = False,
+    ) -> None:
+        span.set_tag_str(SPAN_KIND, "workflow" if is_workflow else "embedding")
+        span.set_tag_str(MODEL_NAME, span.get_tag(MODEL) or "")
+        span.set_tag_str(MODEL_PROVIDER, span.get_tag(PROVIDER) or "")
+
+        input_tag_key = INPUT_VALUE if is_workflow else INPUT_DOCUMENTS
+        output_tag_key = OUTPUT_VALUE
+
+        try:
+            if is_workflow:
+                if isinstance(input_texts, str) or (
+                    isinstance(input_texts, list) and all(isinstance(text, str) for text in input_texts)
+                ):
+                    formatted_inputs = self.format_io(input_texts)
+                    formatted_str = (
+                        formatted_inputs
+                        if isinstance(formatted_inputs, str)
+                        else json.dumps(self.format_io(input_texts))
+                    )
+                    span.set_tag_str(input_tag_key, formatted_str)
+            elif isinstance(input_texts, str):
+                span.set_tag_str(INPUT_DOCUMENTS, json.dumps(Document(text=str(input_texts))))
```

Comment:
```suggestion
                span.set_tag_str(INPUT_DOCUMENTS, json.dumps([Document(text=str(input_texts))]))
```
I believe input documents should be always a list of documents